src/drt/src/pa/FlexPA_unique.cpp

```diff
@@ -167,17 +168,26 @@ void UniqueInsts::addUniqueInst(frInst* inst)
   }
 
   // Fills data structure that relate a instance to its unique instance
-  UniqueInsts::InstSet& unique_family
+  UniqueInsts::InstSet& unique_class
       = master_orient_trackoffset_to_insts_[inst->getMaster()][orient][offset];
-  if (unique_family.empty()) {
+  inst_to_class_[inst] = &unique_class;
+
+  frInst* unique_inst = nullptr;
+  bool new_unique_class = false;
+  if (unique_class.empty()) {
```

Comment:
```suggestion
  bool new_unique_class = unique_class.empty();
  if (new_unique_class) {
```
Just a suggestion for readability. Feel free to ignore.